ray/pyproject.toml

```diff
@@ -0,0 +1,61 @@
+[build-system]
+requires = [
+    "hatchling>=0.13.0",
+    "setuptools>=66; python_version > '3.0'",
```

Comment:
Only required for editable installations on Python 2

```suggestion
```

Reply:
Does it mean we can drop it from the templates? https://github.com/DataDog/integrations-core/pull/15766